doc/classes/Input.xml

```diff
@@ -418,6 +688,23 @@
 				[b]Note:[/b] For macOS, vibration is only supported in macOS 11 and later.
 			</description>
 		</method>
+		<method name="step_joy_motion_calibration">
+			<return type="void" />
+			<param index="0" name="device" type="int" />
+			<description>
+				Saves the current state of the specified joypad's motion sensors readings for the calibration process. The values added into a special list that will be used to adjust the motion sensors readings.
+				For more precision, this method should be called at least 10 times in the calibration process so the engine can adjust the motion sensors readings based on at least 10 different values, otherwise a warning will be printed.
```

Comment:
```suggestion
				For more precision, this method should be called at least 10 times in the calibration process so the engine can adjust the motion sensor readings based on at least 10 different values, otherwise a warning will be printed.
```